src/ansys/dpf/core/operators/averaging/elemental_difference.py

```diff
@@ -14,6 +14,12 @@
 from ansys.dpf.core.config import Config
 from ansys.dpf.core.server_types import AnyServerType
 
+# For type checking
+from ansys.dpf.core.field import Field
```

Comment:
@moe-ad could we put that in an `if TYPE_CHECKING` using `from typing import TYPE_CHECKING` just to make sure it will not create import issues down the line?